packages/tools/locales/nl_NL.po

```diff
@@ -3966,7 +3963,7 @@ msgstr "N"
 
 #: packages/app-cli/app/app.js:137
 msgid "y"
-msgstr "j"
+msgstr "J"
```

Comment:
The source string has lowercase `y`. Why do you use an uppercase character for the translation?